scripts/dcos-net-agent-setup.ps1

```diff
@@ -16,37 +14,32 @@ $variables = (Resolve-Path "$PSScriptRoot\variables.ps1").Path
 
 $TEMPLATES_DIR = Join-Path $PSScriptRoot "templates"
 
-
 function Install-VCredist {
```

Comment:
I see that you removed the `Install-VCredist` calls from below.

You install these earlier in `DCOSWindowsAgentSetup.ps1` script, which makes sense.

But right now, the function `Install-VCredist` from `dcos-net-agent-setup.ps1` is not used anymore and it can be safely removed.

Reply:
I will clean this up